charts/rasa-x/values.yaml

```diff
@@ -183,10 +187,20 @@ rasa:
   jaegerSidecar: "false"
   livenessProbe:
     enabled: true
+    # liveness probe path
+    path: "/status"
```

Comment:
The /status URI requires authentication, not sure if we should have this as a default.